vendor/github.com/coreos/etcd/client/keys.go

```diff
@@ -658,7 +656,7 @@ func unmarshalHTTPResponse(code int, header http.Header, body []byte) (res *Resp
 
 func unmarshalSuccessfulKeysResponse(header http.Header, body []byte) (*Response, error) {
 	var res Response
-	err := codec.NewDecoderBytes(body, new(codec.JsonHandle)).Decode(&res)
+	err := jsoniter.ConfigFastest.Unmarshal(body, &res)
```

Comment:
need to use stdlib-compatible one that is case-sensitive, etc

Reply:
ack switching to `jsoniter.ConfigCompatibleWithStandardLibrary`

Reply:
actually, ConfigCompatibleWithStandardLibrary is not case-sensitive, and we *really* want it to be (c.f. https://github.com/kubernetes/kubernetes/pull/65034, https://github.com/kubernetes/kubernetes/issues/64612)

Reply:
for comparison, ugorji is case-sensitive

Reply:
Ack @liggitt Let's do this in https://github.com/etcd-io/etcd/pull/10667/commits and then bring back the changes here for testing.